kopf/clients/auth.py

```diff
@@ -82,22 +82,36 @@ def login_client(verify=False):
 def verify_pykube():
     """
     Verify if login has succeeded, and the access configuration is still valid.
+
+    All other errors (e.g. 403, 404) are ignored: it means, the host and port
+    are configured and are reachable, the authentication token is accepted,
+    and the rest are authorization or configuration errors (not a showstopper).
     """
     try:
         api = get_pykube_api()
         rsp = api.get(version="", base="/")
         rsp.raise_for_status()
+        api.raise_for_status(rsp)  # replaces requests's HTTPError with its own.
```

Comment:
I think the `rsp.raise_for_status()` can be removed, because `api.raise_for_status(rsp)` [already does it](https://github.com/hjacobs/pykube/blob/dcf39979bc336dd36bb5f70f993797c7d19741a5/pykube/http.py#L239-L241) and you are already catching everything.

Basically if the first line raises, the second would as well (but will not be run because the first raised), and if the first line doesn't raise, then the second won't either, so nothing is gained by having both.